modules/vulcan/src/main/scala/fs2/kafka/vulcan/AvroSettings.scala

```diff
@@ -118,6 +119,12 @@ sealed abstract class AvroSettings[F[_]] {
     createAvroSerializerWith: (F[SchemaRegistryClient], Boolean, Map[String, String]) => F[(KafkaAvroSerializer, SchemaRegistryClient)]
     // format: on
   ): AvroSettings[F]
+
+  /**
+    * Includes the credentials properties from the provided [[KafkaCredentialStore]]
+    */
+  def withCredentials(credentialsStore: KafkaCredentialStore): AvroSettings[F] =
+    withProperties(credentialsStore.properties)
```

Comment:
I don't think this is needed here or in SchemaRegistryClientSettings - these don't connect directly to the Kafka cluster.